spec/features/product_page_spec.rb

```diff
@@ -3,13 +3,13 @@
 feature 'Product page' do
   context 'product exists' do
     before do
-      @product = create(:retailer_product, current_price: 1000)
+      @product = create(:product, current_price: 1000)
     end
     scenario "user can navigate to a retailer's product site" do
       visit products_path
       expect(page).to have_content 'Study Desk'
       click_on 'Study Desk'
-      expect(page).to have_link('$1,000.00', href: @product.url)
+      expect(page).to have_content @product.description
```

Comment:
Oh..this scenario is about a user can click on a price link to navigate to external retailer product page, so in this page, should change to 'click on SHOP or BUY button' to get there.